opencti-worker/src/worker.py

```diff
@@ -264,16 +271,17 @@ def start(self) -> None:
                             bundles_processing_time_gauge,
                             self.objects_max_refs,
                         )
-                        execution_pool = push_execution_pool
+                        is_realtime = False
                         if is_priority_connector(connector["connector_priority_group"]):
-                            execution_pool = realtime_push_execution_pool
+                            is_realtime = True
```

Comment:
can be simplified
`is_realtime = is_priority_connector(connector["connector_priority_group"])`